site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -117,7 +117,9 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.controller.healthCheck.failureThreshold`    | Number of times before giving up (in seconds)                                                   | `3`                    |
 | `agones.controller.healthCheck.timeoutSeconds`      | Number of seconds after which the probe times out (in seconds)                                  | `1`                    |
 | `agones.controller.resources`                       | Controller [resource requests/limit][resources]                                                 | `{}`                   |
-| `agones.controller.generateTLS`                     | Set to true to generate TLS certificates or false to provide your own certificates in `certs/*` | `true`                 |
+| `agones.controller.generateTLS`                     | Set to true to generate TLS certificates or false to provide your own certificates              | `true`                 |
```

Comment:
The new lines should be added to the "New Configuration Features" table below, which will be published along with the next release (in about a week). Changes here apply to the live website, and since this feature isn't part of the chart until the next release, we want to hide it until then.